### PR TITLE
Feature/jest

### PR DESCRIPTION
**Pull request by jwermuth**
This PR implements jest as test framework. It replaces PR #46. It is created as part of Issue #9

### CODE REVIEW

**Comment by jwermuth** [comment]
@labnol let me know if there is something I can do to help you merging this PR

File: README.md [hunk 1]
```diff
@@ -78,6 +78,29 @@ Create a new repository in Github and make a note of the URL of the new reposito
...
+npm run test
+```
+This has limitations:
+
+* You _can_ test code that has no dependencies to Google App Script code, e.g.
+```
+const hasCpuTime = () => !(Date.now() - START_TIME > ONE_MINUTE * 4);
+
+```
+* You _can not_ test code that has no dependencies to Google App Script code, e.g.
```
Comment (labnol) [inline README.md:93]: Please change "has no dependencies" to "has dependencies"

**Comment by labnol** [comment]
@jwermuth Can you please update the pull request using the latest version of package.*json files. Thanks.

File: README.md (hunk 1 above)
Comment (jwermuth) [reply README.md]: Done e2b6bb2

Comment (labnol) [reply README.md]: Just stumbled upon [this repo](https://www.npmjs.com/package/gasmask) that generates stubs for testing Apps Script services locally.

**Comment by jwermuth** [comment]
Updated to lastest in 678d009 . Please advice if committing generated changes from ```npm install``` is not the correct approach.

File: README.md (hunk 1 above)
Comment (jwermuth) [reply README.md]: Interesting. I'll mention #9. Perhaps someone (me?) will have some time to try this out.

**Pull request merged**